src/Symfony/Component/HttpKernel/Profiler/ProfilerStorageInterface.php

```diff
@@ -29,11 +29,12 @@ interface ProfilerStorageInterface
     /**
      * Finds profiler tokens for the given criteria.
      *
-     * @param int|null $limit The maximum number of tokens to return
-     * @param int|null $start The start date to search from
-     * @param int|null $end   The end date to search to
+     * @param int|null    $limit       The maximum number of tokens to return
+     * @param int|null    $start       The start date to search from
+     * @param int|null    $end         The end date to search to
+     * @param string|null $virtualType The type of virtual request (a profile with a virtual type is hidden by default unless this argument is defined)
      */
-    public function find(?string $ip, ?string $url, ?int $limit, ?string $method, int $start = null, int $end = null): array;
+    public function find(?string $ip, ?string $url, ?int $limit, ?string $method, int $start = null, int $end = null/* , ?string $virtualType = null */): array;
```

Comment:
I'd rather keep this interface as-is and have the new parameter hidden, for internal use

Reply:
Fixed.